Description: Fix qmake makefile generation - should not add double slashes to makefiles.
Author: Sune Vuorela <debian@pusling.com>

--- a/qmake/generators/makefile.cpp
+++ b/qmake/generators/makefile.cpp
@@ -2433,7 +2433,7 @@ MakefileGenerator::writeSubTargets(QText
 
         QString mkfile = subtarget->makefile;
         if(!in_directory.isEmpty())
-            mkfile.prepend(out_directory);
+            mkfile.prepend(out_directory + Option::dir_sep);
 
         QString in_directory_cdin, in_directory_cdout, out_directory_cdin, out_directory_cdout;
 #define MAKE_CD_IN_AND_OUT(directory) \
